### ISSUE TITLE
[SUGGESTION] WZS: Suggestion and Balancing

### ISSUE THREAD

**Issue by TheFray89** [Azurblau/Zombies.Zone#9]
- Add a stock for each weapon (High Tier) Example: Vulcan can only be purchased by 2 people, Ivan from 2, Bulldog from 2, and the others, so that the whole team can be balanced, and thus have different weapons , but above all, to give variety to the game (since now people only buy bulldogs).

- Buff the 20% scratcher of life and damage would be a really good thing since its really a very weak and unused zombie.

- Buff the Mummy Zombie of 15% damage, his hp is perfect and interesting at the moment.

- Add a mechanic that can make sure that according to the count of the players that are in the server at that moment, will let there be more zombie bosses.

- Adding the bosses in an Objective Map, it would also give more difficulty to the gameplay (since its extremely easy for humans to win any map), Now I'll give you an example: every 6-7 minutes (based on the map and its size) will spawn a completely random zombie boss.

- Add the command "votekick" for bouncers, because unfortunately, they will never solve the problem of people who troll on purpose.

**Comment by jems01** [comment]
Personally I think the scratcher already is very powerful, I've been knocked down and killed by them many times.

Concerning bosses for obj maps, I discussed this with Jack and we reached the conclusion that bosses such as the shade would be completely overpowered, instead, we combated this issue by using a human dmg coefficient. Suggestions for the multiplier are welcome, and of course, mention the map.

Concering more bosses, I've recently finished a boss script which will add 2x the amount of bosses (exclusive of playercount), for easy maps such as mlt defense. This script can of course be added to, for example, having 2x bosses only when there's a certain playercount, or a boss on a certain wave depending on the playercount. Again, suggestions are super welcome for those.

Last but not least, I'll talk to Jack about votekick, since I'm for that.
